pkg/smtpclient/smtpclient.go

```diff
@@ -67,14 +84,22 @@ func (creds *SMTPCredentials) sendMailAuthSMTP(recipient, subject, body string)
 	return nil
 }
 
-func (creds *SMTPCredentials) sendMailSMTP(recipient, subject, body string) error {
+func (creds *SMTPCredentials) sendMailSMTP(recipient, subject, message string) error {
 
 	c, err := smtp.Dial(fmt.Sprintf("%s:%d", creds.SMTPServer, creds.SMTPPort))
 	if err != nil {
 		return err
 	}
 	defer c.Close()
 
+	host, err := os.Hostname()
```

Comment:
`os.Hostname()` may come to problems when running inside k8s containers, we would be probably better off with a configurable sender hostname (like it is done e.g. in [gomail](https://github.com/go-gomail/gomail/blob/master/smtp.go#L33-L35))

Reply:
Or maybe as a fallback, we could use domain part of the `SenderMail` as hostname?

Reply:
@mirekys makes a lot of sense. I'll make the change